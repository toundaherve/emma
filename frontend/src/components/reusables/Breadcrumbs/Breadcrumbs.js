import React from "react";

const Breadcrumbs = () => {
  return (
    <div
      class="breadcrumbs row"
      itemscope=""
      itemtype="http://schema.org/BreadcrumbList"
    >
      <div class="col12">
        <span
          itemprop="itemListElement"
          itemscope=""
          itemtype="http://schema.org/ListItem"
        >
          <span itemscope="" itemtype="http://schema.org/Thing" itemprop="item">
            <a href="/menu" itemprop="name" content="Menu">
              Menu
            </a>
            <meta itemprop="url" content="/menu" />
          </span>
          <meta itemprop="position" content="1" />
        </span>
        <span class="arrow">›</span>
        <span
          itemprop="itemListElement"
          itemscope=""
          itemtype="http://schema.org/ListItem"
        >
          <span itemscope="" itemtype="http://schema.org/Thing" itemprop="item">
            <a
              href="/menu/big-mouth-burgers"
              itemprop="name"
              content="Big Mouth Burgers®"
            >
              Big Mouth Burgers®
            </a>
            <meta itemprop="url" content="/menu/big-mouth-burgers" />
          </span>
          <meta itemprop="position" content="2" />
        </span>
        <span class="arrow">›</span>
        <span
          itemprop="itemListElement"
          itemscope=""
          itemtype="http://schema.org/ListItem"
        >
          <span itemscope="" itemtype="http://schema.org/Thing" itemprop="item">
            <strong itemprop="name">Queso Burger*</strong>
            <meta
              itemprop="url"
              content="/menu/big-mouth-burgers/queso-burger"
            />
          </span>
          <meta itemprop="position" content="3" />
        </span>
      </div>
    </div>
  );
};

export default Breadcrumbs;
