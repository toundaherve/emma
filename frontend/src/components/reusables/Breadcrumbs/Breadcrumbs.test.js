import React from "react";
import renderer from "react-test-renderer";

import Breadcrumbs from "./Breadcrumbs";

it("renders correctly", () => {
  const component = renderer.create(<Breadcrumbs />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
