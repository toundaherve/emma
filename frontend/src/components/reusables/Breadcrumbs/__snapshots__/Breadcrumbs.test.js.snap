// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  class="breadcrumbs row"
  itemscope=""
  itemtype="http://schema.org/BreadcrumbList"
>
  <div
    class="col12"
  >
    <span
      itemprop="itemListElement"
      itemscope=""
      itemtype="http://schema.org/ListItem"
    >
      <span
        itemprop="item"
        itemscope=""
        itemtype="http://schema.org/Thing"
      >
        <a
          content="Menu"
          href="/menu"
          itemprop="name"
        >
          Menu
        </a>
        <meta
          content="/menu"
          itemprop="url"
        />
      </span>
      <meta
        content="1"
        itemprop="position"
      />
    </span>
    <span
      class="arrow"
    >
      ›
    </span>
    <span
      itemprop="itemListElement"
      itemscope=""
      itemtype="http://schema.org/ListItem"
    >
      <span
        itemprop="item"
        itemscope=""
        itemtype="http://schema.org/Thing"
      >
        <a
          content="Big Mouth Burgers®"
          href="/menu/big-mouth-burgers"
          itemprop="name"
        >
          Big Mouth Burgers®
        </a>
        <meta
          content="/menu/big-mouth-burgers"
          itemprop="url"
        />
      </span>
      <meta
        content="2"
        itemprop="position"
      />
    </span>
    <span
      class="arrow"
    >
      ›
    </span>
    <span
      itemprop="itemListElement"
      itemscope=""
      itemtype="http://schema.org/ListItem"
    >
      <span
        itemprop="item"
        itemscope=""
        itemtype="http://schema.org/Thing"
      >
        <strong
          itemprop="name"
        >
          Queso Burger*
        </strong>
        <meta
          content="/menu/big-mouth-burgers/queso-burger"
          itemprop="url"
        />
      </span>
      <meta
        content="3"
        itemprop="position"
      />
    </span>
  </div>
</div>
`;
